brave/brave-ios
# IPFS Nft Gateway setting

Resolves https://github.com/brave/brave-ios/issues/6835


## Summary of Changes


This pull request fixes #<number>

## Submitter Checklist:


## Test Plan:
New IPFS Nft gateway setting is added.
1) Check that all requests related to the nft wallet page are going to the correct gateway.
2) Check that error is shown if new url is not IPFS gateway or has wrong format.
3) Check that gateway changes correctly.

## Screenshots:

![Simulator Screen Shot - iPhone 14 Pro - 2023-02-02 at 01 46 47](https://user-images.githubusercontent.com/106654560/216174676-b7191f97-c2cf-4f68-8423-5add343feb98.png) | ![Simulator Screen Shot - iPhone 14 Pro - 2023-02-02 at 01 46 55](https://user-images.githubusercontent.com/106654560/216174702-394f46b8-a65c-4c37-b79d-f5ab1114c67a.png) | ![Simulator Screen Shot - iPhone 14 Pro - 2023-02-02 at 01 47 03](https://user-images.githubusercontent.com/106654560/216174736-f5d70301-e753-4cb7-8c51-72ab027ac9a9.png) | ![Simulator Screen Shot - iPhone 14 Pro - 2023-02-02 at 01 48 39](https://user-images.githubusercontent.com/106654560/216174789-e4c0b253-d757-46b5-8228-95cee92369c4.png)
--|--|--|--

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
